js/components/addressBookList/addressBookList_controller.js

```diff
@@ -7,6 +7,14 @@ angular.module('contactsApp')
 	AddressBookService.getAll().then(function(addressBooks) {
 		ctrl.addressBooks = addressBooks;
 		ctrl.loading = false;
+		if(ctrl.addressBooks.length === 0) {
+			AddressBookService.create(t('contacts', 'Contacts')).then(function() {
```

Comment:
You're too demanding! 😢 

Let move this to the next milestone, I can't put my head into coding tonight, too fuzzy! 😄